services/apps/src/apps/routes.clj

```diff
@@ -1,4 +1,4 @@
-(ns apps.routes.api
+(ns apps.routes
```

Comment:
The `apps.routes.api` namespace was renamed to `apps.routes` for consistency with our other services.